chia/rpc/wallet_rpc_api.py

```diff
@@ -3599,3 +3608,41 @@ class VCRevoke(Streamable):
         return {
             "transactions": [tx.to_json_dict_convenience(self.service.config) for tx in txs],
         }
+
+    async def crcat_approve_pending(self, request) -> Dict:
+        """
+        Moving any "pending approval" CR-CATs into the spendable balance of the wallet
+        :param request: Required 'wallet_id'. Optional 'min_amount_to_claim' (deafult: full balance).
+        Standard transaction params 'fee' & 'reuse_puzhash'.
+        :return: all relevant 'transactions'
```

Comment:
could maybe be clearer on what 'relevant transactions' means here

Reply:
Added clarification